fabric_am/test/test_network_handler.py

```diff
@@ -1060,14 +1060,15 @@ def test_PortMirror(self):
         #
         sliver = NetworkServiceSliver()
         # service name (set by user) - only guaranteed unique within a slice
-        sliver.set_name('PortMirror-UKY')
+        sliver.set_name('PortMirror-RENC')
         sliver.set_type(ServiceType.PortMirror)
         sliver.set_layer(NSLayer.L2)
 
         # mirror_port is the name of the port being mirrored - actual name the way
         # service definition needs it. It comes directly from ASM network service sliver
         # whatever the right name is - user must to know it when creating a slice
-        sliver.mirror_port = "TwentyFiveGigE0/0/0/24"
+        #sliver.mirror_port = "HundredGigE0/0/0/17"
+        sliver.mirror_port = "HundredGigE0/0/0/17.100"
         # direction also comes from ASM network service sliver
         sliver.mirror_direction = MirrorDirection.Both
 
```

Comment:
`sliver.mirror_port` would still be `HundredGigE0/0/0/17`
`sliver.mirror_vlan` would now contain  vlan tag `100` which should be used.

Reply:
Changed to sliver.mirror_vlan.